src/AutoRest.CSharp.V3/JsonRpc/Message.cs

```diff
@@ -14,118 +16,4 @@ internal class Message : IMessage
-    }
-
-    internal class ArtifactMapping : IArtifact
-    {
-        public string? Uri { get; set; }
-        public string? Type { get; set; }
-        public string? Content { get; set; }
-        public Mapping[]? SourceMap { get; set; } = null;
-
-        public override string ToString() => $@"{{""uri"":""{Uri}"",""type"":""{Type}"",""content"":""{Content.ToStringLiteral()}""{SourceMap.TextOrEmpty($@",""sourceMap"":{SourceMap.ToJsonArray()}")}}}";
-    }
-
-    internal class RawSourceMap
-    {
-        public string? File { get; set; } = null;
-        public string? SourceRoot { get; set; } = null;
-        public string? Version { get; set; }
-        public string[]? Sources { get; set; }
-        public string[]? Names { get; set; }
-        public string[]? SourcesContent { get; set; } = null;
-        public string? Mappings { get; set; }
-
-        public override string ToString() => $@"{{""version"":""{Version}"",""sources"":{Sources.ToJsonArray()},""names"":{Names.ToJsonArray()},""mappings"":""{Mappings}""{File.TextOrEmpty($@",""file"":""{File}""")}{SourceRoot.TextOrEmpty($@",""sourceRoot"":""{SourceRoot}""")}{SourcesContent.TextOrEmpty($@",""sourcesContent"":{SourcesContent.ToJsonArray()}")}}}";
-    }
-
-    internal class Mapping
-    {
-        public Position? Generated { get; set; }
-        public Position? Original { get; set; }
-        public string? Source { get; set; }
-        public string? Name { get; set; } = null;
-
-        public override string ToString() => $@"{{""generated"":{Generated},""original"":{Original},""source"":""{Source}""{Name.TextOrEmpty($@",""name"":""{Name}""")}}}";
-    }
-
-    // The Channel that a message is registered with.
-    internal enum Channel
-    {
-        // Information is considered the mildest of responses; not necessarily actionable.
-        Information,
-        // Warnings are considered important for best practices, but not catastrophic in nature.
-        Warning,
-        // Errors are considered blocking issues that block a successful operation.
-        Error,
-        // Debug messages are designed for the developer to communicate internal AutoRest implementation details.
-        Debug,
-        // Verbose messages give the user additional clarity on the process.
-        Verbose,
-        // Catastrophic failure, likely aborting the process.
-        Fatal,
-        // Hint messages offer guidance or support without forcing action.
-        Hint,
-        // File represents a file output from an extension. Details are a Artifact and are required.
-        File,
-        // Content represents an update/creation of a configuration file. The final URI will be in the same folder as the primary config file.
-        Configuration,
-        // Protect is a path to not remove during a clear-output-folder.
-        Protect
-    }
```

Comment:
It is fine to separate these into different files. In development, it was much easier to have everything in the same file since these are basically POCOs with 1 method. So, they are all related to each other. You do lose that cohesiveness when going to separate files. Maybe put them into the same namespace?

Reply:
They are in the same `AutoRest.CSharp.V3.JsonRpc` namespace

Reply:
What I meant is... everything was in `Message.cs` which was already in `AutoRest.CSharp.V3.JsonRpc`. So, to preserve intent of that, the models that were in `Message.cs` could to be put into a nested namespace like `AutoRest.CSharp.V3.JsonRpc.MessageModels` (keeping them together and not mixed in with all other classes in `AutoRest.CSharp.V3.JsonRpc`).

Reply:
Done